tests/fixtures/controllers/duplicateMethodController.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  Get,
+  Patch,
+  Put,
+  Route,
+} from '../../../src';
+import { ModelService } from '../services/modelService';
+import { TestModel } from '../testModel';
+import { BaseController } from './baseController';
+
+@Route('DuplicateMethodTest')
```

Comment:
I have some trouble following why methods here are duplicate but not for the `InheritanceMethodController` below. I also haven't yet understood if it's allowed to override a method defined in a base class or not. Maybe some documentation could help.

Reply:
The purpose of this PR seems to be to reflect typescript class inheritance into OpenAPI spec, so I think you're supposed to be allowed to override methods.